spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -139,6 +139,8 @@ def select_team
           puts "#{i + 1}) \"#{team['contentProvider']['name']}\" (#{team['contentProvider']['contentProviderId']})"
         end
 
+        raise "Running in non-interactive shell" if UI.interactive? == false
```

Comment:
a `UserInterface` class has been added to _spaceship_ so you can change this line to be 
```
raise  "Running in non-interactive shell" unless UserInterface.interactive?
```